cassandane/Cassandane/Cyrus/Annotator.pm

```diff
@@ -504,9 +504,13 @@ sub test_log_missing_acl
         $self->{store}->set_folder('Other Users.other');
         $self->make_message("Email", body => "set_flag $_->{flag}\r\n",
             store => $self->{store}) or die;
-        my $wantLog = "could not write flag due missing ACL: "
-                    . "flag=<\\$_->{flag}> need_rights=<$_->{need_rights}>";
-        $self->assert_syslog_matches($self->{instance}, qr{$wantLog});
+        my $wantLog = qr{
+            could\snot\swrite\sflag\sdue\smissing\sACL:
+            (?:\s\w+=<[^>]*>)*
```

Comment:
Oh, this won't skip past `t.id=<...>` because `.` is not `\w`!  These will need to be `\s[^=]+=...` probably...